library

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
```

Comment:
I'd recommend running these with `set -euo pipefail`. It'll make your bash more reliable and reduce unintended actions.

Reply:
what does this do?

Reply:
`man bash` for answers